armi/materials/tests/test_air.py

```diff
@@ -232,16 +232,17 @@ def test_massFrac(self):
         refO = 0.210748
         refAR = 0.004671
 
-        nDens = densityTools.getNDensFromMasses(air.pseudoDensity(Tk=300), air.massFrac)
+        nuclides, nDens = densityTools.getNDensFromMasses(air.pseudoDensity(Tk=300), air.massFrac)
```

Comment:
```suggestion
        _nuclides, nDens = densityTools.getNDensFromMasses(air.pseudoDensity(Tk=300), air.massFrac)
```

Reply:
Either this or, better, test the values of `nuclides`. That is currently an unused variable.